src/components/custom/ZwaveGraph.vue

```diff
@@ -752,7 +752,9 @@ export default {
 
         batlev = batlev ? batlev.value : undefined
 
-        const nodeName = node.name || 'NodeID ' + node.id
+        const nodeName =
+          (node.name || 'NodeID ' + node.id) +
+          (node.loc ? '\n(' + node.loc + ')' : '')
```

Comment:
```suggestion
          (node.loc ? ' (' + node.loc + ')' : '')
```

Reply:
@robertsLando I used a line break here to use two lines and prevent the node symbols to get too wide, since the graph is already very wide.
I did not manage to show both lines centered though.

Reply:
This is what mine looks currently - without showing the location, which would make it much wider:
![image](https://user-images.githubusercontent.com/207989/108692062-73e68580-74fc-11eb-848b-e02a51ab22a5.png)

Reply:
Ok ignore this, btw if we will use the popup window this part will be removed, I prefer the info window to the popup